crates/pallet-grandpa-finality-verifier/src/grandpa.rs

```diff
@@ -1,4 +1,5 @@
 use codec::{Decode, Encode};
```

Comment:
I think we need to add copyright header here as well and mention Parity as the original author of the code (if it is mostly copied from there, otherwise we can leave it as is).